### PR TITLE
homework-9  done  using swagger  and debug-toolbar

### PR DESCRIPTION
**Pull request by Temurbtz**

### CODE REVIEW

File: cleaning_system/cleaning_system/settings.py [hunk 1]
```diff
@@ -37,24 +37,33 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
```
Comment (Temurbtz) [inline cleaning_system/cleaning_system/settings.py:40]: added  swagger  and  debug-toolbar

File: cleaning_system/cleaning_system/settings.py [hunk 2]
```diff
@@ -37,24 +37,33 @@
...
     'request',
    'rating_review',
    'notifications',
-  
+    'drf_yasg',
+   
 ]
 
 MIDDLEWARE = [
+     "debug_toolbar.middleware.DebugToolbarMiddleware",
```
Comment (Temurbtz) [inline cleaning_system/cleaning_system/settings.py:54]: added   debug  toolbar  middleware

File: cleaning_system/cleaning_system/urls.py [hunk 3]
```diff
@@ -23,5 +29,14 @@
     path('',include('cleaning_company.urls')),
     path('',include('rating_review.urls')),
     path('',include('service_type.urls')),
-     path('',include('request.urls'))
+     path('',include('request.urls')),
+
+
```
Comment (Temurbtz) [inline cleaning_system/cleaning_system/urls.py:34]: swagger  urls

File: cleaning_system/cleaning_system/urls.py [hunk 4]
```diff
@@ -23,5 +29,14 @@
     path('',include('cleaning_company.urls')),
     path('',include('rating_review.urls')),
     path('',include('service_type.urls')),
-     path('',include('request.urls'))
+     path('',include('request.urls')),
+
+ 
+   path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
+   path('redoc/', schema_view.with_ui('redoc', cache_timeout=0), name='schema-redoc'),
+
```
Comment (Temurbtz) [inline cleaning_system/cleaning_system/urls.py:37]: debug  toolbar  urls

File: cleaning_system/cleaning_system/urls.py [hunk 5]
```diff
@@ -1,20 +1,26 @@
...
 from django.contrib import admin
 from django.urls import path,  include
+from rest_framework import permissions
+
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+from  django.conf  import  settings
+import  debug_toolbar
+schema_view = get_schema_view(
```
Comment (Temurbtz) [inline cleaning_system/cleaning_system/urls.py:11]: added  swagger  settings  as  it  was  indicated  in  documentation

**Review by DennTerentyev — APPROVED** [review]

**Pull request merged**